cadence-style-guide.md

```diff
@@ -0,0 +1,65 @@
+# Cadence <- Style Guide
+## Table of contents
+1. [Identation](#identation)
+1. [Line length](#line-length)
+1. [Comments](#comments)
```

Comment:
It's a markdown feature, you just put 1 before each line and it will handle the numeration properly
<img width="392" alt="Captura de Pantalla 2022-07-28 a las 19 42 47" src="https://user-images.githubusercontent.com/6607145/181602965-ec9d79ae-3538-4563-aaf2-54b518a2ef44.png">